composer.json

```diff
@@ -19,7 +19,8 @@
     "require": {
         "php": ">=5.3.0",
         "psr/log": "^1.0",
-        "symfony/var-dumper": "^2.6|^3.0"
+        "symfony/var-dumper": "^2.6|^3.0",
+        "symfony/debug": "2.8.*|3.0.*"
```

Comment:
So if symfony/debug 3.1 is released you don't want it to upgrade?
I think it should be `"symfony/debug": "^2.8|^3.0"`
Take a look at [Composers documentation on version numbers](https://getcomposer.org/doc/articles/versions.md)

If you had created unit tests you didn't import the `Symfony\Component\Debug\Exception\FatalThrowableError` in `DebugBar\DataCollector\ExceptionsCollector` in the first place without the requirement of symfony/debug